.github/workflows/release-drafter.yml

```diff
@@ -1,5 +1,19 @@
-# Config for https://github.com/apps/release-drafter
-branches:
-  - master
-template: |
-  $CHANGES
+name: Release Drafter
```

Comment:
@efiop could you take a look at stuff related to releases please?

Reply:
according to https://github.com/marketplace/actions/release-drafter#usage, it seems that "standard" .github/workflows/release-drafter.yml is needed and .github/release-drafter.yml is the second file with main configuration for draft creation https://github.com/marketplace/actions/release-drafter#configuration